privaterelay/context_processors.py

```diff
@@ -12,11 +12,13 @@ def common(request):
     fxa = _get_fxa(request)
     avatar = fxa.extra_data['avatar'] if fxa else None
     accept_language = request.headers.get('Accept-Language', 'en-US')
+    country_code = request.headers.get('x_client_region', 'us')
```

Comment:
@groovecoder unless there's something changing `_` to `-`, the header name will come in as `x-client-region`

Reply:
Thanks. I saw `x_client_region` in the GCP log explorer, so I figured that's what it would be. Can we tell if it will be all-lowercase like this too?

Reply:
@groovecoder yea I think that's more of a GCP log thing, but as long as they are [following RFC](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2) these headers should be case-insensitive.